### ISSUE TITLE
[BITV] 9.4.1.2/7.2 - The "Change color" button in the "Background" section opens an area where you can choose a background color.  (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41904]
However, it does not communicate to users whether the area is open or closed, for example, using the "aria-expanded" attribute. Additionally, the labeling of colored radio buttons has been implemented incorrectly, as the "aria-label" attribute is placed on the "label" element instead of the "input" element.

![image](https://github.com/nextcloud/server/assets/6078378/7e355394-36d2-41fb-83a4-15e76c1fd94e)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v7-n2

**Comment by ShGKme** [comment]
The labelling issue is to be fixed by https://github.com/nextcloud-libraries/nextcloud-vue/pull/4889

`aria-expanded` is a common `NcPopover` issue, mentioned in other issues.

**Comment by susnux** [comment]
waiting for release

**Comment by szaimen** [comment]
> `aria-expanded` is a common `NcPopover` issue, mentioned in other issues.

aria-expanded doesnt seem to be here still...

**Comment by ShGKme** [comment]
Indeed...

**Comment by szaimen** [comment]
backport to stable28 seems to be missing:
![image](https://github.com/nextcloud/server/assets/42591237/2756626f-1edd-4cc2-bcc7-f3aecef97d6e)
